microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -185,87 +188,87 @@ To use the HTML editor:
 
    For example, try adding `<p>Dijon</p>` after the `<p>Marseille</p>` line.
 
-1. Type **Ctrl+Enter** to apply the changes to the DOM tree and close the HTML editor.
+1. Press **Ctrl+Enter** to apply the changes to the DOM tree and close the HTML editor:
```

Comment:
obscure.  is there a GUI way?  a right-click menuitem?  some alternative to memorizing a keyboard shortcut that's more complex/non-standardized than Enter or Ctrl+F?